### DISCUSSION TITLE
IBM PS/2 Model 30 with 8086 CPU

### DISCUSSION THREAD

**Discussion by pioneerman2k1** [86Box/86Box#4030] [Ideas]
Hi everyone, I'm new to this board, although I've been using 86Box for a long time.

I have an IBM PS/2 Model 30 with 8086 CPU in my basement. Unfortunately I could not find this configuration in the emulator. Is it possible to include it in the list? Do you need a BIOS for this? Could this possibly be read out.

**Comment by ghost** [comment]
I believe it's on hold due to its internal MCGA-based graphics.

**Reply by pioneerman2k1** [reply]
That could be. However, at that time it was possible to use an 8 bit ISA VGA graphics card. Just ignore the internal graphics unit for now?

**Comment by Fenix770** [comment]
Hello! 

I'd like to post a request for other machine here, because it seems to have the same Faraday FE2011 chipset that the PS/2 Model 30 8086.
The machine is the Olivetti PCS 86. 

It included a NEC V30 8 Mhz CPU, 1.44mb floppy disk drive, an internal XTA IDE controller and, I think is a unique XT, because it included two SIMM memory modules. So the memory can be expanded and used as LIM EMS, and  Paradise VGA  graphics card. 

Here is some basic info. (Not sure if it shares some fuctions with FE2010 chipset) 
https://www.dosdays.co.uk/media/faraday/FE2011.pdf

Here are the EMS driver and some screenshots
[PCS86.zip](https://github.com/user-attachments/files/16406624/PCS86.zip)
